.github/workflows/pr-close-delete-env.yml

```diff
@@ -5,25 +5,61 @@ on:
     types: [closed]
   workflow_dispatch:
     inputs:
-      environment:
-        description: "Pull request number"
+      slotID:
+        description: "Pull request number (Add multiple IDs seperated by a space)"
         required: true
         default: ""
+  workflow_call:
+    inputs:
+      slotIDs:
```

Comment:
Use a json serialized array instead of space delimited values. That way you can use fromJson on it when defining the matrix

Reply:
recommend json array and not space delimited (and apply to other slotIds param)